server/routes/visitors.ts

```diff
@@ -254,22 +261,20 @@ export default function routes(
       const errors = validationResult(req)
 
       if (!errors.isEmpty()) {
-        return res.render('pages/additionalSupport', {
-          errors: !errors.isEmpty() ? errors.array() : [],
-          offenderNo,
-          additionalSupportOptions: additionalSupportOptions.items,
-          additionalSupportRequired: req.body.additionalSupportRequired,
-          additionalSupport: req.body.additionalSupport,
-          otherSupportDetails: req.body.otherSupportDetails,
-        })
+        req.flash('errors', errors.array() as [])
+        req.flash('formValues', req.body)
+        return res.redirect(req.originalUrl)
       }
 
       let selectedSupport
```

Comment:
Could rearrange this to make it a `const`, the default value would be what you're doing in the else?